README.md

```diff
@@ -1,75 +1,50 @@
-`devel` | Extends the corresponding CUDA image to add the full RAPIDS build and test toolchain (gcc, build tools, etc.) to the system and/or `rapids` environment as well as the notebooks, their dependencies, and runs a Jupyter server as the default Docker `ENTRYPOINT` | Users that are doing active development on RAPIDS and need to build and test their changes
+This image can be found here: https://hub.docker.com/r/rapidsai/rapidsai
 
-At a high-level, the differences between `base`, `runtime`, and `devel` is the way RAPIDS is installed.  `base` and `runtime` are identical in how RAPIDS is installed, with the only difference between them is that `runtime` has (many) more 3rd-party packages installed to support the notebooks.  `devel` is completely different in that RAPIDS is built from source in the container and installed into the `rapids` environment using an install command.  Because of these differences, we often refer to the images as `base & runtime` and `devel`.
+It contains the basic installation of RAPIDS and [`dask-sql`](https://github.com/dask-contrib/dask-sql). By default it starts an `ipython` REPL.
 
-### Image Locations
+### Notebooks image
 
-RAPIDS releases both stable and nightly images in the following repositories. `stable` releases match our `conda` stable version releases. While our `nightly` releases are generated every night from the latest WIP development branch. Below is a table of their repositories and tag lists:
+This image can be found here: https://hub.docker.com/r/rapidsai/rapidsai-notebooks
 
-Type | `stable` Repository | `nightly` Repository
---- | --- | ---
-`base` | [`rapidsai/rapidsai`](https://hub.docker.com/r/rapidsai/rapidsai/tags?page=1&name=base) | [`rapidsai/rapidsai-nightly`](https://hub.docker.com/r/rapidsai/rapidsai-nightly/tags?page=1&name=base)
-`runtime` | [`rapidsai/rapidsai`](https://hub.docker.com/r/rapidsai/rapidsai/tags?page=1&name=runtime) | [`rapidsai/rapidsai-nightly`](https://hub.docker.com/r/rapidsai/rapidsai-nightly/tags?page=1&name=runtime)
-`devel` | [`rapidsai/rapidsai-dev`](https://hub.docker.com/r/rapidsai/rapidsai-dev/tags) | [`rapidsai/rapidsai-dev-nightly`](https://hub.docker.com/r/rapidsai/rapidsai-dev-nightly/tags)
+It extends the `base` images to include RAPIDS notebooks and a [`jupyterlab` server](https://jupyter.org/) which starts automatically.
 
-## Extending Images
+## Image tags
 
-Like any Docker image, the RAPIDS images can be extended to suit the needs of individual teams. Whether it is to add custom libraries, change security settings, or other customizations; using `FROM` and our RAPIDS images allows users to customize the container, but easily update to the latest versions with a new `docker build`.
+Tags for both `base` and `notebooks` images take the form of `${RAPIDS_VER}-cuda${CUDA_VER}-${LINUX_VER}-py${PYTHON_VER}`.
 
-### Custom Token Example
+There is no `latest` tag.
 
```

Comment:
Ah. I see now that we don't want to support a `latest` tag.

We should delete the existing `latest` tag or consider adding `latest` tag support to resolve https://github.com/rapidsai/docker/issues/504.

Reply:
I plan to delete it after the release.